Dockerfile.test

```diff
@@ -18,6 +18,7 @@ FROM node:16 as js_builder
 RUN apt update && apt install -y gcc g++ make libcairo2-dev libjpeg-dev libgif-dev libpango1.0-dev
 
 COPY weave-js /root/weave-js
+COPY wb_schema.gql /root/wb_schema.gql
```

Comment:
this line is setup for later. it allows us to set [this config](https://github.com/wandb/weave/pull/674/files#diff-7224807c4c6115ced2bd4a96644f02b5820ca82b00461fcee63122d73c073d78R5) to `../wb_schema.gql` instead of `./wb_schema.gql`

we'll also need to make a similar change in core [here](https://github.com/wandb/core/blob/b0e9c470f6ee5ca0730c5ff013d8e2810100fde2/services/weave-python/Dockerfile#L6)